test/handlers/ConfirmationControllerRequestHandlerSpec.scala

```diff
@@ -237,4 +237,8 @@ class ConfirmationControllerRequestHandlerSpec extends SpecBase with CoreTestDat
     (loadResultData(manyPeriods).value.asInstanceOf[PhaseOneConfirmationDataResult]).confirmationViewBreakdown.furlough.total mustBe 2402.63
   }
 
+  "do phase two calculation if claimStartDate is on or after 1 July 2020" in new ConfirmationControllerRequestHandler {
+    loadResultData(phaseTwoJourney()).value.isInstanceOf[PhaseTwoConfirmationDataResult] mustBe true
```

Comment:
Using a matcher for the type should give better error messages were this to fail:
```suggestion
    loadResultData(phaseTwoJourney()).value mustBe a[PhaseTwoConfirmationDataResult]
```